### PR TITLE
Update mono_repo generated output

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
Feel free to squash/merge and delete the branch if you're happy

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
-# Created with package:mono_repo v3.1.0-beta.2
+# Created with package:mono_repo v3.1.0-beta.3
 name: Dart CI
 on:
   push:
     branches:
-      - $default-branch
+      - main
```
Comment (natebosch) [inline .github/workflows/dart.yml:6]: What is this doing?

Comment (kevmoo) [reply .github/workflows/dart.yml:6]: `$default-branch` is only valid for **templates** – learned that the hard way

Defaulting these to master & main to cover the common default cases.

Unlikely someone will have both. Eliminates the need for a lot of custom logic/config

Comment (natebosch) [reply .github/workflows/dart.yml:6]: I don't understand, isn't that what this file is? The existing config is working...

Comment (natebosch) [reply .github/workflows/dart.yml:6]: Synced off-thread. LGTM.

**Review by natebosch — APPROVED** [review]

**Pull request merged**